examples/using-email-channel/app.php

```diff
@@ -46,5 +48,8 @@
     json_decode($_ENV[NotifierInput::JIRA_USERS_TO_IGNORE], true)
 ));
 
+$loader = new FilesystemLoader(__DIR__ . '/templates');
+$twig = new Environment($loader);
+
 $output = new NotifierOutput(new EchoOutput());
-$output->write($result);
+$output->write($result, $twig);
```

Comment:
Do not pass is as another argument to the write function. Pass it to the `NotifierOutput` constructor, for example. What do you think?

Reply:
I thought it but it is used only on this method, that is why I prefer to pass it through the write method.
But contextual and lexically is a better option to pass the object by the constructor.